docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -153,6 +153,381 @@ $ quarkus-project-0.1-SNAPSHOT-runner -XX:PrintFlags=
   -XX:±VerboseGC                               Print more information about the heap before and after each collection. Default: - (disabled).
 ----
 
+[[rss]]
+=== Resident Set Size (RSS)
+
+As described in the
+xref:performance-measure.adoc[Measuring Performance guide],
+the footprint of Quarkus applications is measured using the resident set size (RSS).
+This is also applicable to native applications,
+but the runtime engine that manages the footprint is GraalVM's SubstrateVM rather than the JVM.
+
+The reporting techniques specified in the
+xref:performance-measure.adoc[Measuring Performance guide]
+are applicable to native applications too,
+but what causes the RSS to be higher or lower is specific to how GraalVM's `native-image` works.
```

Comment:
If you decide to use https://github.com/quarkusio/quarkus/pull/31061/files#r1106734381 this might also fit better here:

```suggestion
but what causes the RSS to be higher or lower is specific to how the generated native executables work.
```